diem-move/diem-framework/DPN/releases/artifacts/current/build/DPNFramework/BuildInfo.yaml

```diff
@@ -165,7 +165,7 @@ compiled_package_info:
     ? address: "00000000000000000000000000000001"
       name: XUS
     : DiemFramework
-  source_digest: 6CD98BF93B0DD3287029A2EF9C788D5C7871FF13B79D2F17256BB42885A39C3C
+  source_digest: 7DA6FB50576D6D1C277228DD430480010DCE130D7E1F2CFDC35AF0135D8E4F06
```

Comment:
I am curious why this changes? I thought that the fixed `digest.rs` will just ignore non-.move files, thus producing the same digest as before. Perhaps, this PR changes the order of processing the .move files which perhaps affects the final digest value?

Reply:
ah! good question. I was concerned about this as well, so did some digging: The DPNFramework package has no Move sources, but it does have a `README` file in it. So the hash is changing since that README file is no longer being included in the source hash.